dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/datadeposit/UrlManagerServiceBean.java

```diff
@@ -0,0 +1,203 @@
+package edu.harvard.iq.dataverse.api.datadeposit;
+
+import edu.harvard.iq.dataverse.util.SystemConfig;
+import org.apache.commons.lang.StringUtils;
+import org.swordapp.server.SwordError;
+import org.swordapp.server.UriRegistry;
+
+import javax.ejb.Stateful;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Logger;
+
+@Stateful
+public class UrlManagerServiceBean {
```

Comment:
I'm not sure if we should seperate UrlManagerServiceBean and UrlManager atm. They are still dirty and I think it should be a separate task. You can just use @ Stateful at previous class.